src/server/scripts/Northrend/Naxxramas/boss_loatheb.cpp

```diff
@@ -82,10 +82,10 @@ class boss_loatheb : public CreatureScript
             void EnterCombat(Unit* /*who*/) override
             {
                 _EnterCombat();
-                events.ScheduleEvent(EVENT_NECROTIC_AURA, 17 * IN_MILLISECONDS);
-                events.ScheduleEvent(EVENT_DEATHBLOOM, 5 * IN_MILLISECONDS);
-                events.ScheduleEvent(EVENT_SPORE, RAID_MODE(36,18) * IN_MILLISECONDS);
-                events.ScheduleEvent(EVENT_INEVITABLE_DOOM, 2 * MINUTE * IN_MILLISECONDS);
+                events.ScheduleEvent(EVENT_NECROTIC_AURA, Seconds(17));
+                events.ScheduleEvent(EVENT_DEATHBLOOM, Seconds(5));
+                events.ScheduleEvent(EVENT_SPORE, Seconds(18));
```

Comment:
Intended.
First spore is 18 seconds on both modes. Intervals afterwards differ depending on raid mode.
`- First spore now spawns properly after 18 seconds on 10-man (down from 36 seconds). Time between subsequent spawns is unchanged.`